### PR TITLE
doc: Added basic packaging doc.

### PR DESCRIPTION
**Pull request by ashcrow**

### CODE REVIEW

File: doc/packaging.rst [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+Move Source Distribution To RPM Source
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. note::
+
+   Your rpmbuild root may be different! Check with your distribution.
+
+.. code-block:: shell
+
+   (virtualenv)$ mv dist/commissaire-*.tar.gz ~/rpmbuild/SOURCES/
```
Comment (mbarnes) [inline doc/packaging.rst]: One awkward thing about having the source URL pointing to GitHub archives is the tarball files omit the project name (e.g. `0.0.1rc1.tar.gz`), whereas `./setup.py sdist` includes it (e.g. `commissaire-0.0.1rc1.tar.gz`).

I don't know if we can correct for that discrepancy in `setup.py` or on GitHub, but as it stands now you need to strip off `commissaire-` from the tarball name for `rpmbuild`.

**Comment by mbarnes** [comment]
Looks good apart from the tarball name issue above.  :+1:

File: doc/packaging.rst (hunk 1 above)
Comment (ashcrow) [inline doc/packaging.rst]: Updated

**Comment by mbarnes** [comment]
Oh, even better!  Yep, that works.

**Comment by mbarnes** [comment]
Squashed and merged.

**Pull request closed without merging**